pyodide_build/pypabuild.py

```diff
@@ -191,6 +209,45 @@ def _build_in_isolated_env(
             )
 
 
+def _build_in_current_env(
+    build_env: Mapping[str, str],
+    srcdir: Path,
+    outdir: str,
+    distribution: Literal["sdist", "wheel"],
+    config_settings: ConfigSettingsType,
+    skip_dependency_check: bool = False,
+) -> str:
+    with common.replace_env(build_env):
+        # Setup sysconfigdata path in environment
+        sysconfigdata_name = get_build_flag("SYSCONFIG_NAME")
+        sysconfig_dir = Path(get_build_flag("TARGETINSTALLDIR")) / "sysconfigdata"
+        env = os.environ.copy()
+        env["_PYTHON_SYSCONFIGDATA_NAME"] = sysconfigdata_name
+        env["PYTHONPATH"] = f"{str(sysconfig_dir)}:{env.get('PYTHONPATH', '')}"
+
+        with common.replace_env(env):
```

Comment:
I guess `build_env` created from `get_build_env` would already have these (`_PYTHON_SYSCONFIGDATA_NAME`, `PYTHONPATH`) values in the map. Why does this need to re-calculated?

Reply:
> Why does this need to re-calculated?

It doesn't; this was an oversight. Thanks for the catch!

Reply:
Addressed in 8530171ac2f30bbb18099089382cffb89ca7c51d.